### DISCUSSION TITLE
[GitHub] Your personal access token is about to expire

### DISCUSSION THREAD

**Discussion by ISchwandner** [IBAHCM/RPiR#447] [Archive] [answered]
Hi @IBAHCM/rpir-instructors ,

I got this email form GitHub saying that my personal access token will expire in 7 days:

Hi @ISchwandner,

We noticed your personal access token "DESCRIBE THE TOKEN'S USE CASE" with gist, repo, user, and workflow scopes will expire in 7 days.

If this token is still needed, visit https://github.com/settings/tokens/743753803/regenerate to generate an equivalent.

If you run into problems, please contact support by visiting https://github.com/contact?tags=dotcom-accounts

Thanks,
The GitHub Team

Is that something to worry about and what do I need to do about it?

Many thanks,

Imo

**Comment by richardreeve** [comment, marked as answer]
I think what has happened is that you made a PAT that only lasted for 30 days - this is the default - rather than making it run indefinitely. You'll need to run `usethis::create_github_token()` again to create a new GitHub token for your work. I would suggest having no expiration this time for simplicity. And then run `gitcreds::gitcreds_set()` and paste in the new token and run `usethis::edit_r_environ()` and copy in the token again. Clearer(!) instructions are at https://ibahcm.github.io/RPiR/articles/pages/install_github.html#add-a-pat

**Reply by ISchwandner** [reply]
Many thanks! That's resolved now.
